### PR TITLE
Re-licensing post

### PR DESCRIPTION
**Pull request by batpigandme**
Draft of post describing our re-licensing process.

### CODE REVIEW

**Review by tracykteal — APPROVED** [review]
Looks good to me.

File: content/blog/relicensing-packages/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+[^5]: "Prior art" includes the re-licensing of the Bootstrap framework, the details of which are nicely documented in this [StackExchange thread](https://opensource.stackexchange.com/questions/6097/how-does-bootstrap-v4-mit-deal-with-contributions-made-under-v3-apache-2-0/6099#6099).
+
+You can see an example of the process in [the re-licensing issue for purrr](https://github.com/tidyverse/purrr/issues/805).
+The re-licensing generated some discussion but we were grateful to receive unanimous agreement to re-license, thus avoiding the need to re-implement any existing code[^6].
+
+[^6]: This is permitted under what is known as the "idea-expression" dichotomy in copyright law, codified in 17 U.S.C. § 102 under which "protection is given only to the expression of the idea-not the idea itself" *Mazer v. Stein*, 347 U.S. 201 (1954) at 217.
+
+The bulk of our packages are now under MIT, which means they're consistent (yay!), and you can continue to use them as you were before (especially since we didn't think there was any problem using them for any reason under their previous licenses).
+
+This blog post has been a long time coming gives a very reductive summaries of nuanced topics.
```
Comment (juliasilge) [inline content/blog/relicensing-packages/index.Rmd:70]:
```suggestion
This blog post has been a long time coming and gives a very reductive summaries of nuanced topics.
```
???

**Review by batpigandme — COMMENTED** [review]
Good changes—I used the visual editor to get the footnotes working, and it made some things kinda wonky.

**Review by hadley — APPROVED** [review]
Looks go to go to me!

**Pull request merged**